scripts: make test stricter

joshmoore opened the pull request:
After the tests were merged but *before* the implementation was
merged, the lastest-ci run of the tests still passed. With this
change merged, latest-ci should go red (though perhaps that isn't
necessary.

see: https://github.com/ome/omero-py/pull/186#issuecomment-608287585

joshmoore committed:
scripts: make test scricter

After the tests were merged but *before* the implementation was
merged, the lastest-ci run of the tests still passed. With this
change merged, latest-ci should go red (though perhaps that isn't
necessary.

see: https://github.com/ome/omero-py/pull/186#issuecomment-608287585

joshmoore commented:
Finally convinced travis to be green!

jburel commented:
https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-test-integration/lastCompletedBuild/testReport/OmeroPy.test.integration.scriptstest.test_cli/TestCLI/testCLI/ is not passing, cf. comment in description

joshmoore committed:
scripts: use onecmd

cli.invoke() closes the client after execution. onecmd
*doesn't* but it also doesn't take the `strict` param.
Instead, use `assertRC()` to check the return code.

pwalczysko commented:
scricter -> stricter ? (header of this PR)

joshmoore commented:
Done.

jburel commented:
https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-test-integration/372/ is now green
